client/src/pages/blog/components/likedislike.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import { AiFillLike, AiFillDislike } from 'react-icons/ai';
+
+const LikeDislike = (props) => {
+
+    const [likeClicked, setLikeClicked] = useState(false);
+    const [dislikeClicked, setDisikeClicked] = useState(false);
+
+    const handleLikeClick = async (event) => {
+        if (!likeClicked && !dislikeClicked) {
+            try {
+                const response = await fetch(`${process.env.NEXT_PUBLIC_BACKEND_ENDPOINT}/blog/like/${props.blogId}`);
+                if (response.ok) {
+                    setLikeClicked(true);
+                }
+            } catch (error) {
+                console.log('exception in like');
+            }
+        }
+    }
+
+    const handleDislikeClick = async (event) => {
+        if (!likeClicked && !dislikeClicked) {
+            try {
+                const response = await fetch(`${process.env.NEXT_PUBLIC_BACKEND_ENDPOINT}/blog/dislike/${props.blogId}`);
+                if (response.ok) {
+                    setDisikeClicked(true);
+                }
+            } catch (error) {
+                console.log('exception in like');
+            }
+        }
+    }
+
+  return (
+    <div style={{'display': 'flex', 'justify-content': 'flex-end'}}>
```

Comment:
May be an extra Feedback variable could help. I think something like in the Figma design.